concourse/pipelines/deploy.yml

```diff
@@ -223,6 +223,7 @@ resources:
     source:
       expression: "0 06 * * 1-5"
       location: "Europe/London"
+      fire_immediately: true
```

Comment:
Why does `everyday-at-6am` need fire_immediately but not `everyday-at-11pm`?

Reply:
![Screenshot 2021-04-16 at 11 39 47](https://user-images.githubusercontent.com/43747728/115016476-fd875380-9eac-11eb-9cf2-f573c818dd48.png)
 This is the issue we have.

Reply:
it's not a problem for the 11pm one as it is stand-alone task so it is ok to wait 11pm for a version to be created

Reply:
Should we mark the input as `optional: true` as well (or instead) do you think? I don't think it makes any difference 🤔.

Reply:
it doesn't work on its own